lens/base/transform.rkt

```diff
@@ -7,10 +7,15 @@
   (require rackunit
            fancy-app))
 
-(provide lens-transform
-         lens-transform*)
+(provide
+ lens-transform*
```

Comment:
Why not provide a contract for `lens-transform*`?
At least a
```racket
(->* [any/c] #:rest (listof (or/c lens? (-> any/c any/c))) any/c)
```
 would be better than nothing, right?

Also, if you wanted to use my `match-string` package you could do this:
```racket
(->* [any/c] #:rest (append/c (list/c lens? (-> any/c any/c)) '...) any/c)
```
http://pkg-build.racket-lang.org/doc/match-string/index.html#%28def._%28%28lib._match-string%2Fmain..rkt%29._append%2Fc%29%29